data/sql/db-world/zone_difficulty_hardmode_ai.sql

```diff
@@ -0,0 +1,31 @@
+-- ************************************************************************
+-- Possible values for `Target`:
+SET @TARGET_SELF = 1;                       -- Self cast.
+SET @TARGET_VICTIM = 2;                     -- Our current target. (ie: highest aggro)
+SET @TARGET_HOSTILE_SECOND_AGGRO = 3;       -- Second highest aggro.
+SET @TARGET_HOSTILE_LAST_AGGRO = 4;         -- Lowest aggro.
```

Comment:
Maybe you will use them later? Atm these are unused

Reply:
Yes, they're for future use. There is no example yet for these targets.